layouts/index.html

```diff
@@ -25,7 +25,7 @@
           <div id="skip-target" class="tablet:grid-col-8 right-shift">
             <section class="register align-bottom">
               <header>
-                <h1 class="main-heading" data-test="main-header">{{ $translation.homepage.header | safeHTML }}</h1>
+                <h1 class="main-heading" data-test="main-header">{{ $translation.homepage.header }}</h1>
```

Comment:
@SamiraMSadat Please revert this file change.

Reply:
Fixed